### PR TITLE
Fixes lizard icons in security records and AI holograms.

### PR DESCRIPTION
**Pull request by SvartaSvansen**
No more snoutless, tailless abominations! The title pretty much says it all.

A side effect of the examine icon change is humans having hair. So it affects people on both sides of the species fence.

### CODE REVIEW

**Comment by Jordie0608** [comment]
Why not just use the appearance var?

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -22,7 +22,8 @@
 		t_is = "are"
 	else
 		if(icon)
-			msg += "\icon[src] " //note, should we ever go back to runtime-generated icons (please don't), you will need to change this to \icon[icon] to prevent crashes.
+			var/icon/nude_icon = data_core.get_id_photo(src, 1) //Scandalous.
+			msg += "\icon[nude_icon] "
```
Comment (MrPerson) [inline code/modules/mob/living/carbon/human/examine.dm]: This bug should be fixed in the current byond beta. Still gotta test that at some point

**Comment by SvartaSvansen** [comment]
I have fixes I'm trying to sync, but I don't have time for it right now. I'll work on this some more when I can.

Real life and all.

**Comment by Shadowlight213** [comment]
@svartasvansen you still working on this?

**Comment by Razharas** [comment]
Over month old with no progress - closed

**Comment by Razharas** [comment]
@phil235 
You can reopen

**Pull request closed without merging**